homeassistant/components/tuya/select.py

```diff
@@ -242,6 +242,12 @@
             icon="mdi:timer-cog-outline",
             translation_key="countdown",
         ),
+        SelectEntityDescription(
+            key=DPCode.MIST,
```

Comment:
What is mist? Is it described somewhere in the API documentation?
Does it have known values?

Reply:
I updated the PR description to answer your questions:

"My fan has a "mist"-feature, that means it has a water tank from which water gets vaporized via ultra-sound and injected into the air stream to cool down the room"
"As this is completely undocumented in the Tuya docs, I implemented it as a simple enum since I don't know what other devices might report"

Reply:
> I implemented it as a simple enum

This ain't an enum?

Reply:
Well, the mist-feature of **my** fan probably should be a number-entity...
As the feature is not documented, I chose an Enum to also support devices having non-numeric values.

What do you recommend?
Switching to a numeric value?!

Reply:
I took another look at it, I suggest to accept the PR as it is because of:
- Tuya returns an Enum, implementing it as an Enum seems to be the safest and most logical way (as possible return values are unknown because this feature is not documented in the API)
- The Tuya-integration does not support creating a NumericEntity out of an Enum
- Other fan-features that are "Enums returned by Tuya" (like "countdown") are implemented exactly the same way it's done in this PR